### ISSUE TITLE
Show date in the log files

### ISSUE THREAD

**Issue by Wanhedaa** [Icecrown-Community/Icecrown-Precious-Bot-Public#97]
Before new Precious was borned, we had dates shown in the log files. It was very helpfull. Now we don't have them, only the runID, the activity, the payment realm, faction and the pot + cut. 
![image](https://user-images.githubusercontent.com/77633500/130328129-3973cee5-9480-454c-844e-9300e8061fb1.png)

I'd like to ask to change it back as it was before  ^^ 

We have it for movements. I think for the boosters would be also helpfull, so we need it on advertiser + booster tab aswell. 
![image](https://user-images.githubusercontent.com/77633500/130328133-272c98e9-9e9f-48fa-9103-637b9130b719.png)

How i imagine:
![image](https://user-images.githubusercontent.com/77633500/130328180-76c0842b-1a41-4097-8156-f11b4ea6e160.png)

**Comment by Dejnaa** [comment]
Yes please, this shouldnt be to hard to make as it was there before

**Comment by Efthymios-Papakostopoulos** [comment]
• Comments
• Special 

Gabor dm me when you touch this

**Comment by Tueri** [comment]
> Gabor dm me when you touch this

Me too :)

Jk, we can put it here - I think someone can be used for a 1-time blast of design on those logs; customer filtering, better layout - someone once made me a mock and it looked pretty hot 🔥 ⏸️ 🍾
